src/main/webapp/resources/js/pages/projects/samples-metadata-import/components/SampleMetadataImportSelectFile.tsx

```diff
@@ -120,20 +130,42 @@ export function SampleMetadataImportSelectFile(): JSX.Element {
         });
         return false;
       }
-      return true;
     },
   };
 
   return (
     <SampleMetadataImportWizard current={0} status={status}>
       <Spin spinning={loading}>
-        {!valid && (
+        {(showEmptyHeaderError || showDuplicateHeaderError) && (
```

Comment:
With the changes made above we can update this section to be:

{uploadValidationErrors.length && (
          <ErrorAlert
            style={{ marginBottom: SPACE_XS }}
            message={i18n("SampleMetadataImportSelectFile.alert.valid.title")}
            description={
              <>
                {i18n(
                  "SampleMetadataImportSelectFile.alert.valid.description.preface"
                )}
                <ul style={{ marginBottom: 0 }}>{uploadValidationErrors}</ul>
                {i18n(
                  "SampleMetadataImportSelectFile.alert.valid.description.postface"
                )}
              </>
            }
          />
        )}

Reply:
Please see a54ec79fd261822b9a8c159c45f7c4db4a1fb00e.